.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 package-lock.json
 luacov.report.out
 luacov.stats.out
+yarn.lock
```

Comment:
Why is this here? Do we even use yarn?

Reply:
I think because I tried to run the linter with yarn, but it works with both yarn and npm